src/main/java/core/levels/LevelLoader.java

```diff
@@ -2,11 +2,26 @@
 
 import com.badlogic.gdx.math.Vector2;
 import com.badlogic.gdx.physics.box2d.World;
+import core.world.EntitySpawner;
+import static core.world.DemoSpawners.*;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class LevelLoader {
 
+    /**
+     * Load level with spawners and map
+     *
+     */
     public static LevelState getLevel1() {
-        LevelState lvl = new LevelState("maps/demo.tmx");
+        // Create enemy spawner list
```

Comment:
should also spawn the Player

Reply:
would need a LevelManager to spawn player, but LevelLoader is the parameter to LevelManager instantiation so i dt think it should have access to it? would what exists in LevelGameplayController right now not be okay?

Reply:
let's take care of this after GameCharacter is properly merged, for now can just be a TODO

(basically we'd set up a Spawner here for the player)